azure/iac/keyvaultkeys.rego

```diff
@@ -0,0 +1,56 @@
+package rule
+
+# https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults/keys
+
+#
+# PR-AZR-0130-ARM
+#
+
+default kv_keys_expire = null
+
+azure_attribute_absence["kv_keys_expire"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.keyvault/vaults/keys"
+    resource.properties.attributes.enabled != false
```

Comment:
checking enabled attribute is unnecessary here. if we need to check enabled state, create a separate rule for that if already not exist.

Reply:
@rezoan  according to this https://gsl.dome9.com/D9.AZU.CRY.12.html we need to check both objects in one policy

Reply:
@shahinshirvani i understand they said it in a generic way which is actually required for rule to be passed but if we can separate those in a different rule in the same file both will execute and give us the same result in more accurate way, also this will give us more control over remediation description writing.
Also if we separate, we may not need to check the enable state for other rules as well for the same resource type.

@farchide thought?

Reply:
@rezoan @shahinshirvani the idea is we need to check the expiry date for the non-expired keys.
it is very unlikely someone set a key in the code and set the enabled value to false. GSL example is for the cloud live resources which is a possible scenario.

The other item we need to check is `KeyRotationPolicyAttributes` it sets the expiration days and satisfies the purpose of this policy as well

Reply:
fixed